pwnlib/tubes/tube.py

```diff
@@ -180,6 +180,7 @@ def _recv(self, numb = None, timeout = default):
 
         # No buffered data, could not put anything in the buffer
         # before timeout.
+        context.buffer_size = numb
```

Comment:
why?

Reply:
Because I have to fix the line `data = self.proc.stdout.read(numb)` where numb value is alway 4096. Then the data is alway asked and never returned. It is an issue because this use to work with select system call on linux but it does not work on windows. I Have to find a way to get back the data and avoid to break the tests.